reference/tools/microsoft/helpers.rst

```diff
@@ -0,0 +1,63 @@
+.. _conan_tools_microsoft_helpers:
+
+conan.tools.microsoft.visual
+=============================
+
+check_min_vs
+---------------
+
+
+.. currentmodule:: conan.tools.microsoft.visual
+
+.. autofunction:: check_min_vs
+
+
+vcvars_command
```

Comment:
Not for being public yet, I think

Reply:
I recall a couple of recipes using it in conan-center-index...

Reply:
Yes, but as you can see, this one doesn't have the ``conanfile`` as first argument. That is a hint that this is not ready to be made public (as it won't be able to use ``conf`` if needed). Lets hide it, so these use cases arises while the migration and we can check them.

Reply:
oh, ok. That's right